docs/sources/administration/image_rendering.md

```diff
@@ -194,3 +194,21 @@ it's available for the Grafana process, e.g.
 ```bash
 export GF_RENDERER_PLUGIN_CHROME_BIN="/usr/bin/chromium-browser"
 ```
+
+### Grafana image renderer plugin and certificate signed by internal CAs
+
+If you are using the Grafana image renderer with a Grafana server that uses a certificate signed by a custom CA (e.g. a company-internal CA) that is not known to the nss trust store shipped with headless chrome, rendering images will fail and you will see messages like this in the Grafana log:
```

Comment:
What does CA stand for? Please define it in the first use. same quession for "nss."

Reply:
From https://en.wikipedia.org/wiki/Certificate_authority:
> In cryptography, a certificate authority or certification authority (CA) is an entity that issues digital certificates

```suggestion
If you are using the Grafana image renderer with a Grafana server that uses a certificate signed by a custom certificate authority (CA), e.g. a company-internal CA, that is not known to the nss trust store shipped with headless chrome, rendering images will fail and you will see messages like this in the Grafana log:
```